### DISCUSSION TITLE
network change in virtual machine

### DISCUSSION THREAD

**Discussion by ZombCry** [Security-Onion-Solutions/securityonion#13145] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Eval

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

12.1 gb

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I changed the virtual machine for a different network but when I want to change the static IP address it won't let me, what command can I do it with, I checked the security onion page and it tells me that with the so-ip-update command but it doesn't work for me What I can do?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by err0r20** [comment]
Zomb,

Changing the IP (or hostname) after installation is not recommended (see: https://docs.securityonion.net/en/2.4/best-practices.html#configuration). If you need to change the IP address, you can just re-run setup.
